source/tests/pt/test_loss_default_pf.py

```
@@ -0,0 +1,308 @@
+            self.start_pref_v,
+            self.limit_pref_v,
+            start_pref_pf=self.start_pref_pf,
+            limit_pref_pf=self.limit_pref_pf,
+            use_default_pf=False,
+        )
+
+        def fake_model():
+            return self.model_pred
+
+        # With find_atom_pref=0.0 and use_default_pf=False, pf loss contribution is zero
+        _, pt_loss_without, pt_more_loss_without = loss_fn(
```

Ruff (0.15.7)

[warning] Unpacked variable `pt_loss_without` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)